.gitignore

```diff
@@ -20,6 +20,7 @@ Icon?
 # Files that might appear on external disk
 .Spotlight-V100
 .Trashes
+_support/
```

Comment:
Just curious what creates this file in the workspace?

Reply:
It's just a local tool in our toolchain, which generates some files that we don't want committed.
If it's a problem to have this entry, we can leave it out. I mostly added it for my own convenience.